### DISCUSSION TITLE
Ability to hide collection and records from specific users

### DISCUSSION THREAD

**Discussion by SKLn-Rad** [firecmsco/firecms#258] [Q&A]
Hey, apologies if this has been asked before; I did a quick look around and couldn't see anything.

Is it currently possible to hide collections from users who don't meet permission checks within the app?
Something akin to a read permission, to prevent being able to see anything at all.

Super useful for us as we want users to only be able to edit their own records in their collections. (Basically content creators adding their own stuff and not being able to see or edit anyone elses)

Thanks!
Ryan

**Comment by fgatti675** [comment]
Hi @SKLn-Rad 
When you define your navigation, you can use a builder based on the logged in user, so you can select the collections that each user should see 
`NavigationBuilder`

**Reply by SKLn-Rad** [reply]
Yeah that bit is fine! Let me rephrase it slightly.

We have clients who will be writing articles on our platform, and want to some of them from seeing entire collections (with the method above), and also block them seeing specific records in collections which were not created by themselves.

**Reply by fgatti675** [reply]
Ah we have this requested already but it is not possible at the moment.
We are including it as a new feature for version 2
